docs/tasks/stateful-sets/deleting-a-statefulset.md

```diff
@@ -0,0 +1,82 @@
+---
+assignees:
+- bprashanth
+- erictune
+- foxish
+- janetkuo
+- smarterclayton
+
+---
+
+{% capture overview %}
+This task shows you how to delete a Stateful Set.
+{% endcapture %}
+
+{% capture prerequisites %}
+
+* If you are not using Stateful Sets, you can skip this task now. 
```

Comment:
Not a good pre-requisite, though it is pretty Google-like.

How about: "This task assumes you have an application running on your cluster represented by a StatefulSet."